.github/workflows/run-archiver.yml

```diff
@@ -1,8 +1,28 @@
 ---
 name: run-archiver
 
+env:
+  SMALL_DATASETS: '"eia176","eia191","eia757a","eia860","eia860m","eia861","eia923","eia930","eiaaeo","eiawater","eia_bulk_elec","epacamd_eia","ferc1","ferc2","ferc6","ferc60","ferc714","mshamines","nrelatb","phmsagas"'
+  LARGE_DATASETS: '"epacems"' # Datasets requiring a large (paid) runner
+
 on:
   workflow_dispatch:
+    inputs:
+      small_runner:
+        description: 'Small runner: Comma-separated list of datasets to archive (e.g., "ferc2","ferc6").'
+        default: '"eia176","eia191","eia757a","eia860","eia860m","eia861","eia923","eia930","eiaaeo","eiawater","eia_bulk_elec","epacamd_eia","ferc1","ferc2","ferc6","ferc60","ferc714","mshamines","nrelatb","phmsagas"'
```

Comment:
While it would be neater, this doesn't like to take an environment variable as an input.

Reply:
Hm, since you're already doing `inputs.small_runner || env.SMALL_DATASETS` at the use site, it seems like you could just let this [default to `""`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callinputs) and then get the desired "default to all small datasets" behavior.